### PR TITLE
Tune Node

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What did you change? \*

- Update to node 6
- Tune node server
- Add **/_v/myorders/enabled** route

#### How to test it? \*

https://sousa--recorrenciacharlie.myvtex.com/_v/myorders/enabled

#### Types of changes \*

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical improvements

### CODE REVIEW

File: node/service.json [hunk 1]
```diff
@@ -1,10 +1,23 @@
 {
   "stack": "nodejs",
-  "memory": 256,
+  "memory": 150,
```
Comment (malvadeza) [inline node/service.json:3]: I'm definitely stealing these.

File: node/index.ts [hunk 2]
```diff
@@ -30,5 +30,9 @@ export default new Service<IOClients>({
       ctx.response.status = 204
       ctx.response.body = 'My Account app is installed in this Store.'
     },
+    enabledMyOrders: (ctx: Context) => {
+      ctx.response.status = 204
+      ctx.response.body = 'My Account app is installed in this Store.'
```
Comment (arthurepc) [inline node/index.ts:35]: Did you mean to respond to both requests with the same message? 
```suggestion
      ctx.response.body = 'My Orders app is installed in this Store.'
```

**Review by arthurepc — APPROVED** [review]
Looking good! I just added a comment, not blocking at all

**Review by malvadeza — APPROVED** [review]
Sweet new configurations 🥇

File: node/index.ts [hunk 3]
```diff
@@ -30,5 +30,9 @@ export default new Service<IOClients>({
       ctx.response.status = 204
       ctx.response.body = 'My Account app is installed in this Store.'
     },
+    enabledMyOrders: (ctx: Context) => {
+      ctx.response.status = 204
+      ctx.response.body = 'My Orders app is installed in this Store.'
```
Comment (rafarubim) [inline node/index.ts:35]: 204 with body? 😛

**Review by rafarubim — APPROVED** [review]

**Pull request merged**